src/h5web/providers/hsds/api.ts

```diff
@@ -82,6 +82,7 @@ export class HsdsApi implements ProviderAPI {
     const childName = path.slice(path.lastIndexOf('/') + 1);
     const child = getChildEntity(parentGroup, childName);
     assertDefined(child);
+    assertHsdsEntity(child);
 
     const entity = isHsdsGroup(child)
```

Comment:
`isHsdsGroup` no longer checks if the entity has an ID. Instead, it expects an HSDS entity.